### DISCUSSION TITLE
如何缩小目录里面 序号与标题之间的距离

### DISCUSSION THREAD

**Discussion by DongdongGui** [nuaatug/nuaathesis#117] [Q&A]
如图所示，请问如何缩小序号与标题之间的距离：
例如， 第一章 与绪论之间的距离 1.1 与研究背景与意义之间的距离

![Clip_2024-05-28_22-41-18](https://github.com/nuaatug/nuaathesis/assets/51312083/2eb43ac6-5b38-45ec-8950-4409389b9901)

**Comment by yzwduck** [comment]
修改 `\titlecontents`.
注意模板把 本科/硕士 以及 中文/英文 的格式写在同一块区域里了, 要修改符合你选择的那个 `\titlecontents`.
文档 https://ctan.org/pkg/titletoc
